React Native v0.64 android build issue

Unable to build android app with RN 0.64

**To Reproduce**
Steps to reproduce the behavior:
1. `npx react-native init stripe_test` # creates new RN project with v0.64
2. Follow steps [here](https://github.com/stripe/stripe-react-native/blob/master/CONTRIBUTING.md#install-library-as-local-repository)
3. `npm run android`

Build Error:

```
error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081
e: /stripe_test/.yalc/stripe-react-native/android/src/main/java/com/reactnativestripesdk/StripeSdkCardView.kt: (20, 76): Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type UIManagerModule?

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':stripe-react-native:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s

    at makeError (/stripe_test/node_modules/execa/index.js:174:9)
    at /stripe_test/node_modules/execa/index.js:278:16
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async runOnAllDevices (/stripe_test/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/runOnAllDevices.js:94:5)
    at async Command.handleAction (/stripe_test/node_modules/react-native/node_modules/@react-native-community/cli/build/index.js:186:9)
info Run CLI with --verbose flag for more details.
```
---
Possibly related to this change: https://github.com/react-native-community/releases/blob/master/CHANGELOG.md#android-specific
>Fix ReadableArray null annotations. Possibly breaking change for Kotlin apps. (d76556543f by @dulmandakh)

--- a/android/src/main/java/com/reactnativestripesdk/StripeSdkCardView.kt
+++ b/android/src/main/java/com/reactnativestripesdk/StripeSdkCardView.kt
@@ -13,11 +13,11 @@ import com.stripe.android.view.CardInputWidget
 class StripeSdkCardView(private val context: ThemedReactContext) : FrameLayout(context) {
   private var mCardWidget: CardInputWidget
   private val cardDetails: MutableMap<String, Any> = mutableMapOf("number" to "", "cvc" to "", "expiryMonth" to "", "expiryYear" to "", "postalCode" to "")
-  private var mEventDispatcher: EventDispatcher
+  private var mEventDispatcher: EventDispatcher?
 
   init {
     mCardWidget = CardInputWidget(context);
-    mEventDispatcher = context.getNativeModule(UIManagerModule::class.java).eventDispatcher
+    mEventDispatcher = context.getNativeModule(UIManagerModule::class.java)?.eventDispatcher
 
     addView(mCardWidget)
     setListeners()
@@ -47,7 +47,7 @@ class StripeSdkCardView(private val context: ThemedReactContext) : FrameLayout(c
 
   fun onCardChanged() {
     val complete = mCardWidget.cardParams != null
-    mEventDispatcher.dispatchEvent(
+    mEventDispatcher?.dispatchEvent(
       CardChangedEvent(id, cardDetails, mCardWidget.postalCodeEnabled, complete))
   }
 
@@ -58,7 +58,7 @@ class StripeSdkCardView(private val context: ThemedReactContext) : FrameLayout(c
       override fun onCvcComplete() {}
       override fun onFocusChange(focusField: CardInputListener.FocusField) {
         if (mEventDispatcher != null) {
-          mEventDispatcher.dispatchEvent(
+          mEventDispatcher?.dispatchEvent(
             CardFocusEvent(id, focusField.name))
         }
       }

